x-pack/legacy/plugins/security/public/views/management/edit_role/components/privileges/kibana/space_aware_privilege_section/__snapshots__/space_aware_privilege_section.test.tsx.snap

```diff
@@ -5,6 +5,7 @@ exports[`<SpaceAwarePrivilegeSection> with user profile disabling "manageSpaces"
   color="danger"
   data-test-subj="userCannotManageSpacesCallout"
   iconType="alert"
+  size="m"
```

Comment:
The snapshot test which were updated are failing in CI. I believe that your version of EUI is likely out of date, and if you run `yarn kbn bootstrap` you'll see these snapshot tests fail locally again.